src/html/preferences.html

```diff
@@ -78,7 +78,11 @@ <h3 data-i18n="products"></h3>
                 <input id="mobile" name="chooseProduct" type="checkbox" value="mobile">
                 <span data-i18n="product_mobile"></span>
             </label>
-            <label for="ios">
+            <label for="firefox-preview" class="checkbox">
+                <input id="firefox-preview" name="chooseProduct" type="checkbox" value="firefox-preview">
+                <span data-i18n="product_firefox-preview"></span>
+            </label>
+            <label for="ios" class="checkbox">
```

Comment:
I think you accidentally readded the checkbok when rebasing your branch. Screenshot of what it looks like when I run it locally

![image](https://github.com/mozillabrasil/sumo_live_helper/assets/7339076/7af46bf9-1df4-4088-926e-c8b321ca4deb)

Reply:
That's what I get for using the GitHub website instead of the desktop application 😆